content/customize-gnome.md

```diff
@@ -44,6 +44,8 @@ This extension allows for the desktop to be themed in further ways than default.
 
 ![dash-to-dock](/images/customize-gnome/dash-to-dock.png)
 
+ >**Note:** Starting with Pop!\_OS 22.04 Cosmic Dock is forked from Dash to Dock. Gnome Settings has options for customization in Settings -> Desktop -> Dock. You will want to disable Cosmic Dock in the Extension application if adding Dash to Dock.
```

Comment:
>Starting with Pop!\_OS 22.04 Cosmic Dock is forked from Dash to Dock.
 
 would this sound better?
 
 >Starting with version 22.04, Pop!\_OS includes Cosmic Dock, which is forked from Dash to Dock.

Saying that Cosmic Dock is a fork since 22.04 doesn't really convey that it's included with the OS, so I'm trying to make this sentence a little more elegant and to-the-point.

Reply:
Fixed in new commit.